Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceStateMachine+State.swift

```diff
@@ -131,6 +131,13 @@ extension ExperienceStateMachine.Transition {
         _ experience: Experience, _ stepIndex: Experience.StepIndex, _ package: ExperiencePackage, _ stepRef: StepReference
     ) -> Self {
         guard let newStepIndex = stepRef.resolve(experience: experience, currentIndex: stepIndex) else {
+            if stepRef == .offset(1) && experience.stepIndices.last == stepIndex {
+                return .init(
+                    toState: .endingStep(experience, stepIndex, package),
+                    sideEffect: .continuation(.endExperience(markComplete: true))
```

Comment:
technically markComplete not required right? since this would get standard handling of "ended on last step so it was complete".  Only thing that caught me eye slightly different than android, but no issue i think!

Reply:
The Action require the parameter and the SDK in other places sets it according to the expected behaviour